config.toml

```
@@ -149,6 +149,13 @@ weight = 11
 [languages.fr.params]
 description = "Le projet de Glossaire CNCF Cloud Native a pour objectif de référencer les termes courants utilisés dans les différentes applications Cloud Natives."
 
+[languages.ru]
+title = "Глоссарий Cloud Native"
+languageName ="Русский (Russian)"
+weight = 12
```

There is no main page for `ru`. 
https://deploy-preview-2468--cncfglossary.netlify.app/ru/

I think contentDir is essential parameter. :)

```suggestion
contentDir = "content/ru"
weight = 12
```

---

Good catch, thanks a lot.